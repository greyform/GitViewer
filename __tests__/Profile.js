require('react-native');
const React = require('react');
const Profile = require('../src/screens/Profile');
const ProfilePage = require('../src/screens/Profile');
// Note: test renderer must be required after react-native.
const renderer = require('react-test-renderer');

it('renders correctly', () => {
  const tree = renderer.create(
    'Profile'
  );
});

it('renders correctly ProfilePage', () => {
  const tree = renderer.create(
    'ProfilePage'
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly HeaderPage', () => {
  const tree = renderer.create(
    'Header'
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly Content', () => {
  const tree = renderer.create(
    'Content'
  ).toJSON();
  expect(tree).toMatchSnapshot();
});


it('renders correctly Bar', () => {
  const tree = renderer.create(
    'Content'
  ).toJSON();
  expect(tree).toMatchSnapshot();
});







/*jest.mock('../src/screens/', () => {
  const mockComponent = require('react-native/jest/mockComponent');
  return mockComponent('path/to/MyNativeComponent');
});*/