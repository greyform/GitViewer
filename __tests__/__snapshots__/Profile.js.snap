// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Bar 1`] = `"Content"`;

exports[`renders correctly Content 1`] = `"Content"`;

exports[`renders correctly HeaderPage 1`] = `"Header"`;

exports[`renders correctly ProfilePage 1`] = `"ProfilePage"`;
